### DISCUSSION TITLE
Providing custom stream

### DISCUSSION THREAD

**Discussion by Riztazz** [ArthurSonzogni/FTXUI#319] [Q&A] [answered]
Hello,

Would it be possible to add a custom stream to write into? Something along the lines of this:
https://github.com/Riztazz/FTXUI/commit/df4701b1144c620dbf65638ced8e7542d6f1268e

Or this will cause some issues down the road?

**Comment by ArthurSonzogni** [comment, marked as answer]
Yes, why not.What is your use case exactly?

PR are welcome if you want to submit it.
Just one request: could you use a new method instead of the constructor?

**Reply by Riztazz** [reply]
No longer needed, my approach was a bit bad, finally figured out how to use ftxui properly:P thanks for great lib!

**Reply by Riztazz** [reply]
A bit out of topic but is there some discord where i could ask questions or should i use discussions/issues?

**Reply by ArthurSonzogni** [reply]
Thanks!
Yes the discussion here is the way to discuss.

There are no discord. Maybe I should, since this is the second time I got this kind of comment. However, I don't see myself opening it regularly.

**Reply by Riztazz** [reply]
Ah i see, i'll just use discussions tab then :)

**Reply by Anubhav-Rai** [reply]
How you did it @Riztazz

**Reply by Riztazz** [reply]
Sorry, i don't remember :(
